pandas/tests/frame/methods/test_set_axis.py

```diff
@@ -24,6 +24,55 @@ def test_set_axis(self, obj):
         result = obj.set_axis(new_index, axis=0, inplace=False)
         tm.assert_equal(expected, result)
 
+    def test_set_axis_copy(self, obj):
```

Comment:
Does this test `inplace=True/copy=False`?

Reply:
yes